pkg/features/features.go

```diff
@@ -150,6 +150,9 @@ const (
 
 	// This option indicates support for HTTPRoute with a backendref with an appProtoocol 'kubernetes.io/ws' (extended conformance)
 	SupportHTTPRouteBackendProtocolWebSocket SupportedFeature = "HTTPRouteBackendProtocolWebSocket"
+
+	// This option indicates support for HTTPRoute filter request redirect (extended conformance)
+	SupportHTTPRouteFilterRequestRedirect SupportedFeature = "HTTPRouteFilterRequestRedirect"
```

Comment:
Since this test is about backend filters, and we already have the `SupportHTTPRoute*Redirect` features in place, I'd suggest to rename this one to `SupportHTTPRouteBackendPathRedirect`. The same renaming should be applied to the tests.